client/helpers.go

```diff
@@ -237,12 +237,12 @@ func getRemoteKey(role data.RoleName, remote store.RemoteStore) (data.PublicKey,
 func rotateRemoteKey(role data.RoleName, remote store.RemoteStore) (data.PublicKey, error) {
 	rawPubKey, err := remote.RotateKey(role)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("unable to rotate remote key: %s", err)
```

Comment:
I don't think we need these because of the `errFmtMsg` in `client/client.go` already prepends this string to the error before passing on